### ISSUE TITLE
Oculus Medium stopped working

### ISSUE THREAD

**Issue by sirkitree** [LibreVR/Revive#784]
Latest update to Oculus Medium seems incompatible. Each time I try to start it I get a dialog:

Error

Unable to initialize the Oculus Rift.

**Comment by Armada651** [comment]
Can you try this version? https://github.com/LibreVR/Revive/releases/tag/1.2.0-beta2

**Comment by sirkitree** [comment]
quill is way smoother too!

**Comment by Armada651** [comment]
Yeah, one of the issues the beta tries to address is poor performance in Oculus Medium and similar apps.
